docker/expose-teams-api.md

```diff
@@ -0,0 +1,31 @@
+<div align="center">
+<p align="center">
+
+<img src="https://user-images.githubusercontent.com/25985824/106288517-2422e000-6216-11eb-871d-26ad2e7b1e59.png" height="55px"> &nbsp;
+<img src="https://user-images.githubusercontent.com/25985824/106288518-24bb7680-6216-11eb-8f10-60052c519586.png" height="50px">
+
+</p>
+</div>
+
+---
+
+# Exposing the `teams-api` service
+
+There are some limited instances in which you may wish to expose your FiftyOne Teams API for SDK access.
```

Comment:
Not limited - it's beta now, but will be the preferred method of SDK connection going forward imo

Reply:
Our conversations suggested that we were not releasing Authenticated API Access as Beta and were going to provide support for specific customers in an early-adopters manner...  Maybe I'm wrong about that...

Got a better phrasing you'd prefer to see?